extensions/wikia/ContentReview/ContentReview.setup.php

```diff
@@ -63,6 +64,11 @@
  */
 $wgExtensionMessagesFiles['ContentReview'] = __DIR__ . '/ContentReview.i18n.php';
 
+/**
+ * Special page
+ */
+$wgSpecialPages['ContentReview'] = 'SpecialContentReviewController';
```

Comment:
Our standard for SpecialPageControllers would be `ContentReviewSpecialController`.